src/index.js

```diff
@@ -4,6 +4,7 @@
 // https://material.angularjs.org/latest/demo/button
 
 // Core
+import { AirbnbRating, Rating } from 'react-native-ratings';
```

Comment:
this is not core, but prettier moves this to the top automatically.

Reply:
Add this on the first line.

`/* eslint-disable import/order */`